### DISCUSSION TITLE
[Collectible Bear] Release date

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2124] [General]
Changed my strategy, invested time and money to reach it and now you guys change the launch date? Why?

I think these unannounced changes cause distrust in the information provided. 
Don't you think about the players? Don't you think about the players' feelings?

You must rethink about it and keep the commitment to the players.

**Comment by adamhannigan** [comment]
Thanks for your feedback @future2back 

This was announced in the game and detailed in our Discord announcements. The Brilliant Bear has not been removed, just delayed.

Thanks for your patience
